templates/al2023/variables-default.json

```diff
@@ -17,6 +18,7 @@
     "kms_key_id": "",
     "iam_instance_profile": "",
     "launch_block_device_mappings_volume_size": "20",
+    "nvidia_major_driver_version": "",
```

Comment:
nit: can we setup default version here instead of Makefile?

Reply:
yes, we can do that

Reply:
Actually, I think we'll need to keep the default version in the Makefile, because we need it in the AMI_VARIANT variable. 

```
nvidia_major_driver_version := $(NVIDIA_MAJOR_DRIVER_VERSION_DEFAULT)
AMI_VARIANT := $(AMI_VARIANT)-$(nvidia_major_driver_version)
```